src/popper/runner_host.py

```diff
@@ -293,6 +293,198 @@ def _update_with_engine_config(self, container_args):
                 container_args[k] = update_with[k]
 
 
+class PodmanRunner(StepRunner):
+    """Runs steps in podman on the local machine."""
+
+    def __init__(self, init_podman_client=True, **kw):
+        super(PodmanRunner, self).__init__(**kw)
+
+        self._spawned_containers = set()
+        _, _, self._p_info = HostRunner._exec_cmd(["podman", "info"], logging=False)
```

Comment:
this has to be moved, otherwise the `init_podman_client` would not be having an effect.

Reply:
I've done this